inferno/callbacks.py

```diff
@@ -162,6 +254,40 @@ def on_batch_end(self, net, X, y, train):
 
 
 class PrintLog(Callback):
+    """Print out useful information from the model's history.
+
+    By default, this will print the epoch, train loss, valid loss, and
+    epoch duration. In addition, `PrintLog` will take care of
+    highlighting the best loss at each epoch.
+
+    To determine the best loss, `PrintLog` looks for a keys that end
+    on `'_best'` and associates them with the corresponding
+    loss. E.g., `'train_loss_best'` will be matched with
+    `'train_loss'`. Therefore, `PrintLog` works best in conjunction
+    with the `BestLoss` callback.
```

Comment:
"Therefore, `PrintLog` works best in conjunction with the `BestLoss` callback which adds these keys automatically." (give reason why it works best in conjunction)